src/js/components/atoms/youtube-link/youtube-link.html

```diff
@@ -1,11 +1,9 @@
-<div class="cta__video">
```

Comment:
This is done so the vertical rythm keeps working as intended. When encapsulating the <a> in some html element the vertical rythm was off, because the other CTA components are just plain <a> tags.

Reply:
i think the vertical rhythm should be fixed instead, but for the efficiency of getting this branche completed - _for another time_